operators/lazy-vs-eager-execution.md

```diff
@@ -0,0 +1,44 @@
+# Lazy vs Eager Execution
+
+The default mode of operator execution is `eager`. This means that whenever an operator is called, it will immediately run. This allows us to provide useful features like type enforcement.
```

Comment:
Let's add a small note saying something like "This is just like executing a regular Python function."